src/containers/Profile.vue

```diff
@@ -0,0 +1,301 @@
+      this.$emit('onboard');
+    },
+    goBack() {
+      // special case: if the user has a graduate program (and not an undergrad program), skip the transfer page
+      if (this.onboarding.grad !== '' && !this.onboarding.college && this.currentPage > 1) {
+        this.currentPage = 1;
+      } else {
+        this.currentPage = this.currentPage - 1 === 0 ? 0 : this.currentPage - 1;
+      }
+    },
+    setPage(page: number) {
+      this.currentPage = page;
+    },
+    canProgress() {
+      return !(this.isError || this.isInvalidMajorMinorGradError);
+    },
+    goNext() {
+      // Only move onto next page if error message is not displayed
+      if (!(this.isError || this.isInvalidMajorMinorGradError)) {
+        // special case: if the user has a graduate program (and not an undergrad program), skip the transfer page
+        if (this.onboarding.grad !== '' && !this.onboarding.college && this.currentPage === 1) {
+          this.currentPage += 2;
+        } else {
+          this.currentPage = this.currentPage === FINAL_PAGE ? FINAL_PAGE : this.currentPage + 1;
+        }
+      }
+    },
+    updateBasic(
+      gradYear: string,
+      gradSem: FirestoreSemesterSeason,
+      entranceYear: string,
+      entranceSem: FirestoreSemesterSeason,
+      college: string,
+      major: readonly string[],
+      minor: readonly string[],
+      grad: string,
+      name: FirestoreUserName
+    ) {
+      this.name = name;
+      this.onboarding = {
+        ...this.onboarding,
+        gradYear,
+        gradSem,
+        entranceYear,
+        entranceSem,
+        college,
+        major,
+        minor,
+        grad,
+      };
+    },
+    // clear transfer credits if the student is only in a graduate program, but previously set transfer credits
+    clearTransferCreditIfGraduate() {
+      if (this.onboarding.grad !== '' && this.onboarding.college === '') {
+        this.updateTransfer([], 'no');
+      }
+    },
+    updateTransfer(exams: readonly FirestoreAPIBExam[], tookSwim: 'yes' | 'no') {
+      const userExams = exams.filter(
+        ({ subject, score }) => score !== 0 && subject !== placeholderText
+      );
+      this.onboarding = {
+        ...this.onboarding,
+        exam: userExams,
+        tookSwim,
+      };
+    },
+    cancel() {
+      if (this.onboardingData.college !== '' || this.onboardingData.grad !== '') {
+        this.$emit('cancelOnboarding');
+      }
+    },
+    checkClickOutside(e: MouseEvent) {
+      if (
+        e.target === this.$refs.modalBackground &&
+        (this.onboardingData.college !== '' || this.onboardingData.grad !== '')
+      ) {
+        this.cancel();
+      }
+    },
+  },
+});
+</script>
+<style scoped lang="scss">
+@import '@/assets/scss/_variables.scss';
+
+.profileContainer {
+  background-color: #f6fafc;
+  padding: 3rem 7rem;
+
+  &-title {
+    width: 523px;
+    height: 40px;
+    right: 753px;
+    top: 40px;
+
+    font-family: 'Proxima Nova';
```

Comment:
This one is not on you - but could you remove this font-family line? We set the font-family globally so it probably doesn't do anything here.